recipe-server/webpack.config.js

```diff
@@ -60,6 +62,9 @@ module.exports = [
         './client/control/sass/control.scss',
         './node_modules/font-awesome/scss/font-awesome.scss',
       ],
+      'console-log': './client/actions/console-log/index',
+      'show-heartbeat': './client/actions/show-heartbeat/index',
+      'preference-experiment': './client/actions/preference-experiment/index',
```

Comment:
What is this? Merging our two webpack configs? Or something else?

Should we remove the other webpack config?